apps/web/playwright/booking-seats.e2e.ts

```diff
@@ -143,6 +144,19 @@ test.describe("Booking with Seats", () => {
       expect(attendeeIds).not.toContain(bookingAttendees[0].id);
     });
 
+    await test.step("Attendee #2 shouldn't be able to cancel booking using only booking/uid", async () => {
+      await page.goto(`/booking/${booking.uid}`);
+
+      await expect(page.locator('[data-testid="cancel"]')).toHaveCount(0);
+    });
+
+    await test.step("Attendee #2 shouldn't be able to cancel booking using randomString for seatReferenceUId", async () => {
+      await page.goto(`/booking/${booking.uid}?seatReferenceUid=${randomString(10)}`);
+
+      // expect cancel button to don't be in the page
+      await expect(page.locator('[data-testid="cancel"]')).toHaveCount(0);
```

Comment:
This test will continue passing if someone updates the component to render a cancel button that doesn't have this exact `data-testid`. What do you think about making this selector more general? e.g. Look for any button with the text `Cancel`

Reply:
Sure!